src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SecurityKeyIdentifierClauseSerializer.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Xml;
+
+namespace CoreWCF.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Abstract class for SecurityKeyIdentifierClause Serializer.
+    /// </summary>
+    public abstract class SecurityKeyIdentifierClauseSerializer
+    {
+        /// <summary>
+        /// When implmented in the derived class will check the element where the reader is
+        /// positioned for a SecurityKeyIdentifierClause type.
+        /// </summary>
+        /// <param name="reader">XmlReader positioned at a SecurityKeyIdentifierClause element.</param>
+        /// <returns>True if the SecurityKeyIdentifierClause can be deserialized.</returns>
+        public abstract bool CanReadKeyIdentifierClause( XmlReader reader );
+
```

Comment:
Formatting of method signatures in file. If you delete the last curly bracket and type it back in, it should fix everything.